interpreter_book/src/Makefile

```diff
@@ -0,0 +1,21 @@
+all: check test race
+
+.PHONY: check
+check:
+	go build ./...
+
+.PHONY: test
+test:
+	go test ./...
+
+.PHONY: race
+race:
+	go test -race ./...
+
+.PHONY: fmt
+fmt:
+	gofmt -s -w ./...
+
+.PHONY: monkey
```

Comment:
Small suggestion: You could have all the "phony"s in one line at the top, we do it at work and I feel it's easier to read the targets that way but non blocking.

Reply:
it's a preference
we have all the phonies in one line at work too, while at the open source project I've been working with we have it like this. I guess, it's easier to remove entries/copy-paste.
Either way, feel free to add a commit on top, I'm very lazy :D

Reply:
good point and thank you for sharing, then lets leave it how you have it :)